tests/pdifftests/test.py

```diff
@@ -26,15 +27,18 @@ class Runs(Base):
     window_size = [1280, 800]
     base_url = BASE
 
+    @hide('span.time')
```

Comment:
Did this not work? Most of the screenshot changes are from "10 days ago"→"1 month ago"